src/Acuminator/Acuminator.Tests/Helpers/EmbeddedFileDataAttribute.cs

```diff
@@ -61,10 +59,11 @@ protected EmbeddedFileDataAttribute(string[] fileNames, string testFilePath, str
 						throw new ArgumentException("File name cannot be empty", nameof(internalCodeFileName));
 				}
 			}
-
+			
 			_prefix = GetPrefixFromTestFilePath(testFilePath);
 
 			_fileNames = externalCodeFileNames.IsNullOrEmpty() ? fileNames :  fileNames.Concat(externalCodeFileNames).ToArray();
+
```

Comment:
Why do you need an empty line before closing brace?

Reply:
Removed